src/app/components/authorized/post/post.component.spec.ts

```diff
@@ -58,6 +58,31 @@ describe('PostComponent', () => {
         expect(component).toBeTruthy();
     });
 
+    it('postId should be 1', () => {
```

Comment:
This entire test suite is pretty much useless. You are providing mocked data and you are testing if the data is really mocked. There is no logical testing in here.

Reply:
I deleted unnecessary code and only left tests to check if getPost method was called from service. Is that alright??